src/main/java/org/sunbird/workflow/service/UserBulkUploadService.java

```diff
@@ -260,27 +244,90 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
+                            }
+                        } else if (nextRow.getCell(11).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.EXTERNAL_SYSTEM_ID, nextRow.getCell(11).getStringCellValue().trim());
+                            if (!ValidationUtil.validateExternalSystemId((String)valuesToBeUpdate.get(Constants.EXTERNAL_SYSTEM_ID))) {
+                                errList.add("Invalid External System ID : External System Id can contain alphanumeric characters and have a max length of 30");
+                            }
+                        } else {
+                            errList.add("Invalid column type. Expecting string/number format");
+                        }
+                    }
+                    if (nextRow.getCell(12) != null && !org.apache.commons.lang.StringUtils.isBlank(nextRow.getCell(12).toString())) {
+                        if (nextRow.getCell(12).getCellType() == CellType.STRING) {
+                            valuesToBeUpdate.put(Constants.EXTERNAL_SYSTEM, nextRow.getCell(12).getStringCellValue().trim());
+                            if (!ValidationUtil.validateExternalSystem((String)valuesToBeUpdate.get(Constants.EXTERNAL_SYSTEM))) {
+                                errList.add("Invalid External System : External System can contain only alphabets and can have a max length of 255");
+                            }
+                        } else {
+                            errList.add("Invalid column type. Expecting string format");
+                        }
+                    }
+                    if (nextRow.getCell(13) != null && nextRow.getCell(13).getCellType() != CellType.BLANK) {
+                        if (nextRow.getCell(13).getCellType() == CellType.STRING) {
+                            String tagStr = nextRow.getCell(13).getStringCellValue().trim();
+                            List<String> tagList = new ArrayList<String>();
+                            if (!org.apache.commons.lang.StringUtils.isEmpty(tagStr)) {
+                                String[] tagStrList = tagStr.split(",", -1);
+                                for(String tag : tagStrList) {
+                                    tagList.add(tag.trim());
+                                }
+                            }
+                            valuesToBeUpdate.put(Constants.TAG, tagList);
+                            if (!ValidationUtil.validateTag((List<String>)valuesToBeUpdate.get(Constants.TAG))) {
+                                errList.add("Invalid Tag : Tags are comma seperated string values. A Tag can contain only alphabets with spaces. eg: Bihar Circle, Patna Division");
+                            }
+                        } else {
+                            errList.add("Invalid column type. Expecting string format");
```

Comment:
Instead of "column" -- do provide which field we are trying to parse and the expected format.

Reply:
Done